### DISCUSSION TITLE
Not receiving email notifications

### DISCUSSION THREAD

**Discussion by uPagge** [umputun/remark42#1391] [Q&A]
Hello
Here's my problem. I set up email notifications for administrators and users. And notifications for admins work great. But notifications for users do not work. Moreover, the token for confirming the subscription comes, but the user does not receive notifications of new comments.
I'm using the latest remark 1.10.1

**Comment by paskal** [comment]
I've just checked, and for me, it works. Can you post the log of sending the notification message? I think of NOTIFY_EMAIL_FROM being different from AUTH_EMAIL_FROM, which could cause only auth emails to go through.
